distributionResources/manual-upgrade.sh

```diff
@@ -70,6 +70,8 @@ print_usage()
     echo ""
     echo "        --tomcat_lk: for LabKey Internal Hosted Server Installed Only"
     echo ""
+    echo "        --skip_tomcat: the script will not attempt to start/stop tomcat.  you must do this manually before/after running the script. This might be suitable for remote pipeline servers as well.
```

Comment:
```suggestion
    echo "        --skip_tomcat: the script will not attempt to start/stop tomcat.  You must do this manually before/after running the script.
```
The mention of a suggestion about remote pipeline servers might only serve to confuse, particularly if it is not a more definite recommendation.

Reply:
Sure, seems like a good idea